knowledge-base/kb-diagram-custom-connection.md

```diff
@@ -0,0 +1,228 @@
+---
+title: Create Custom RadDiagramConnection
```

Comment:
Add more information about the topic. Create Custom RadDiagramConnection is too general. The article shows how to add new connection caps and customize them separately from the connection.